src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -322,6 +323,11 @@ public List<Resource> listResourcesAndSync() {
     return resources;
   }
 
+  /** Populate the list of folders for this workspace */
```

Comment:
the current implementation doesn't populate, it just fetch the folders. but I'm indifferent about whether to store this to the disk